indico/htdocs/sass/modules/_designer.scss

```diff
@@ -352,3 +352,24 @@
 .template-side.back:not(.empty) {
     pointer-events: none;
 }
+
+.toggle-default {
+    opacity: 0.3;
+
+    &.default-on-category {
+        opacity: 1;
+        color: $blue;
+
+        &.inherited-from-root {
+            opacity: 0.6;
+
+            &:hover {
+                color: $blue;
```

Comment:
Isn't the color already blue from `&.default-on-category`? Maybe you wanted to set opacity to 1 instead?